client/src/components/home-page/PageIndex.js

```diff
@@ -3,15 +3,36 @@ import styled from 'styled-components';
 import { useDispatch } from 'react-redux';
 import { useSelector } from 'react-redux';
 import { BsCaretRight } from 'react-icons/bs';
+<<<<<<< HEAD
+import { nextPageItems } from '../../actions';
+
+export default function PageIndex({ page, setPage }) {
+=======
 import { fetchItems } from '../helpers/fetch-functions';
 import { requestItems, receiveItemsInfo } from '../../actions';
 
 export default function PageIndex() {
+>>>>>>> 3c4465f4fe3826ac68989b2b3de57d3b1a19b87e
   const dispatch = useDispatch();
   const items = useSelector((state) => state.items);
-  const lastPage = Math.round(
-    items.totalFound / items.filters.query_result_maxqty
+  const lastPage = Math.round(items.totalFound / items.pageSize);
+  return (
+    <Wrapper>
+      Page {page} of {lastPage}
+      <Button
+        onClick={() => {
+          if (page < lastPage) {
+            setPage((n) => n + 1);
+          }
+          dispatch(nextPageItems());
+        }}
+      >
+        <BsCaretRight color={'gray'} />
+      </Button>
+    </Wrapper>
   );
+<<<<<<< HEAD
+=======
```

Comment:
@garorueda99 
I believe these are from rebasing/fixing conflict. Can you update pl.